serverless-example-python/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+name: Serverless Example Python
+tagline: Sample Hello-World python serverless application
+tags:
+  - Python
+  - Flask
+  - API
+categories:
+  - Examples
+  - Hello-World
+author: Pratik Mahajan <p@phy.cx>
+maintainer: Pratik Mahajan <p@phy.cx>
+
```

Comment:
You could add the logo here as a base64 here itself

Reply:
Thought of keeping it a link, i.e. pushing it to github repo where all files lie rather than having it in base64 because of the following reasons:
1. Database Performance: If we keep it in base64 here, it will also be stored in database. As number of apps increase, the loading time on database might also increase because of images. 
2. Network Performance: If we keep it in database, it will be sent to the browser as a json instead of a link.Thus the response size is increasing in this case. 
3. Web-page performance: if response is big, the web-page will take longer to load. If we are using the link instead, in that case, the web-page will load quickly and image might load according to its convenience.